core/deprecations.py

```diff
@@ -170,9 +170,11 @@ def deprecate_action(time_deprecated: str, name: str):
         id = f"action.{name}.{time_deprecated}"
 
         deprecate_notify(id, f"The `{name}` action is deprecated. See log for more.")
+        replacement_msg = f' Instead, use: "{replacement}".' if replacement else ""
 
         msg = (
             f"The `{name}` action is deprecated since {time_deprecated}."
+            f"{replacement_msg}"
             f' See {os.path.join(REPO_DIR, "BREAKING_CHANGES.txt")}'
```

Comment:
I guess we want an entry in BREAKING_CHANGES.txt?

Reply:
True. Done